CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 ### Fixed
```

Comment:
Oh this is my bad, we don't need this [Unreleased] section. Can you move this into the Fixed section of 3.5-RO? I'll update the date again now that 3/12 is now "today".

Reply:
You got it